lib/routes/zhihu/question.js

```diff
@@ -1,19 +1,37 @@
 const got = require('@/utils/got');
 const utils = require('./utils');
+const config = require('@/config').value;
+const md5 = require('@/utils/md5');
+const g_encrypt = require('./execlib/g_encrypt');
 
 module.exports = async (ctx) => {
+    const cookie = config.zhihu.cookies_no_login;
+    if (cookie === undefined) {
+        throw Error('缺少知乎Cookie 值(不需要登录即可抓取Cookie)'); // the key is "d_c0"
+    }
+
     const { questionId } = ctx.params;
-    const sort = 'created';
+    const sort = 'updated'; // or default,created
     const limit = 20;
-    const include = `data[*].content.excerpt&limit=${limit}&offset=0`;
-    const url = `https://www.zhihu.com/api/v4/questions/${questionId}/answers?include=${include}&sort_by=${sort}`;
+    const include = "data%5B*%5D.is_normal%2Cadmin_closed_comment%2Creward_info%2Cis_collapsed%2Cannotation_action%2Cannotation_detail%2Ccollapse_reason%2Cis_sticky%2Ccollapsed_by%2Csuggest_edit%2Ccomment_count%2Ccan_comment%2Ccontent%2Ceditable_content%2Cattachment%2Cvoteup_count%2Creshipment_settings%2Ccomment_permission%2Ccreated_time%2Cupdated_time%2Creview_info%2Crelevant_info%2Cquestion%2Cexcerpt%2Cis_labeled%2Cpaid_info%2Cpaid_info_content%2Crelationship.is_authorized%2Cis_author%2Cvoting%2Cis_thanked%2Cis_nothelp%2Cis_recognized%3Bdata%5B*%5D.mark_infos%5B*%5D.url%3Bdata%5B*%5D.author.follower_count%2Cbadge%5B*%5D.topics%3Bdata%5B*%5D.settings.table_of_content.enabled&offset=0";
+    const url = `https://www.zhihu.com/api/v4/questions/${questionId}/answers?include=${include}&limit=${limit}&sort_by=${sort}&platform=desktop`;
+
+    // calcuate x-zse-96, see https://github.com/srx-2000/spider_collection/issues/18
+    const pattern = /d_c0=(\S*);/;
+    const cookie_mes = cookie.match(pattern)[1];
```

Comment:
I consider this should be moved to above and add an existence check to prompt users about the lack of `d_c0`.

Reply:
https://github.com/DIYgod/RSSHub/pull/9729/commits/320dd4c070099800f0ceb2f3dcdeadd9e884b99e  yes!